sources/packages/web/src/components/aest/institution/modals/AssessOfferingModal.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-form ref="assessOfferingForm">
-    <modal-dialog-base :showDialog="showDialog" :title="title" max-width="730">
+    <modal-dialog-base :showDialog="showDialog" :title="title" :max-width="730">
```

Comment:
Just as a general idea, we have been adding this `:max-width="730"` in many modals. Maybe this should be set as the default value on the component.

Reply:
No we do not have a ticket for that, mine was adding min-width and max-width in the component which does not have a title in the modal, and we came to a conclusion that we use either one of them.

Reply:
added it in https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1461